index.html

```diff
@@ -566,9 +567,9 @@ <h2 property="schema:name">Authorization Rule</h2>
               <section id="access-objects" inlist="" rel="schema:hasPart" resource="#access-objects">
                 <h3 property="schema:name">Access Objects</h3>
                 <div datatype="rdf:HTML" property="schema:description">
-                  <p id="acl-accessto">The <code>acl:accessTo</code> property value is used to check if access is allowed for a specific resource.</p>
+                  <p id="acl-accessto">The <code>acl:accessTo</code> predicate denotes the resource to which access is being granted.</p>
 
-                  <p id="acl-default">The <code>acl:default</code> property value (the container resource in context) is used to apply the Authorization to the original requested resource.</p>
+                  <p id="acl-default">The <code>acl:default</code> predicate denotes the container resource to which an Authorization can be applied to a resource lower in the collection hierarchy.</p>
```

Comment:
This is not quite English. It's far enough off that I am not sure how to change it. I *think* that the following gets closer to what this is intended to say, but my phrasing is also not optimal.
```suggestion
                  <p id="acl-default">The <code>acl:default</code> predicate denotes the container resource whose Authorization can be applied to a resource lower in the collection hierarchy.</p>
```